src/com/oltpbenchmark/types/DatabaseType.java

```diff
@@ -34,97 +34,46 @@
      * (3) Should SQLUtil.getInsertSQL include col names
      * (4) Does this DBMS support "real" transactions?
      */
-    DB2("com.ibm.db2.jcc.DB2Driver", true, false, true),
-    MYSQL("com.mysql.jdbc.Driver", true, false, true),
-    MYROCKS("com.mysql.jdbc.Driver", true, false, true),
-    POSTGRES("org.postgresql.Driver", false, false, true),
-    ORACLE("oracle.jdbc.driver.OracleDriver", true, false, true),
-    SQLSERVER("com.microsoft.sqlserver.jdbc.SQLServerDriver", true, false, true),
-    SQLITE("org.sqlite.JDBC", true, false, true),
-    AMAZONRDS(null, true, false, true),
-    SQLAZURE(null, true, false, true),
-    ASSCLOWN(null, true, false, true),
-    HSQLDB("org.hsqldb.jdbcDriver", false, false, true),
-    H2("org.h2.Driver", true, false, true),
-    MONETDB("nl.cwi.monetdb.jdbc.MonetDriver", false, false, true),
-    NUODB("com.nuodb.jdbc.Driver", true, false, true),
-    TIMESTEN("com.timesten.jdbc.TimesTenDriver", true, false, true),
-    CASSANDRA("com.github.adejanovski.cassandra.jdbc.CassandraDriver", true, true, false),
-    MEMSQL("com.mysql.jdbc.Driver", true, false, false),
-    NOISEPAGE("org.postgresql.Driver", false, false, true),
+    DB2(true),
+    MYSQL(true),
+    MYROCKS(true),
+    POSTGRES(true),
+    ORACLE(true),
+    SQLSERVER(true),
+    SQLITE(true),
+    AMAZONRDS(true),
+    SQLAZURE(true),
+    ASSCLOWN(true),
+    HSQLDB(true),
+    H2(true),
+    MONETDB(true),
+    NUODB(true),
+    TIMESTEN(true),
+    CASSANDRA(false),
+    MEMSQL(false),
+    NOISEPAGE(true),
```

Comment:
we can probably remove all except postgres.
Maybe we don't need this class itself?

Reply:
i agree, but id prefer to make further removals/deletions in a follow up at this point, given the already massive scope of this change